### ISSUE TITLE
[ERROR REPORT] Jailbreak failed at -77 error

### ISSUE THREAD

**Issue by cnrkuo** [checkra1n/BugTracker#1580]
**Tell us about your setup:**
1. What iDevice are you using?
iPhone 8
2. On what version of iOS is it?
14.0 beta 3
3. What version of checkra1n are you using?
0.10.1 beta
4. What is your host system (OS version? Hackintosh? VM? etc.)?
Linux (bootra1n)
5. How are you connecting to the device (USB-A? USB-C? Apple/3rd party cable? Through a USB hub?)?
USB-A to Lightning

**What are the steps to reproduce the issue?**
1. When I've installed at first, I've selected ¨Allow untested IOS versions." 
2. Start, Go to recovery mode, go  DFU, and -79 error occurred.

**What do you expect, and what is happening instead?**
I've checked Safe Mode, but same......
-79 USBMUX error.

**Does the issue also occur if you tick "Safe Mode" in the checkra1n options?**
Yes, but same error.

**Any other info, error logs, screenshots, ...?**
No

**Comment by demhademha** [comment]
Use and official operating systemlike ubuntu ￼

**Comment by rexsllemel** [comment]
you can also use kali. error -77 referred to locked iDevice. Unlock it first or remove the password. Run on root privilege and every attempt restart the checkrain

**Comment by mattfam0914** [comment]
Samething is happening to me Checkra1n beta 0.10.2 - iPhone X iOS 14 Beta 3; I've done from pi 4. Tried using Ubuntu & Rasperry Pi OS. I've launched via sudo ./checkra1n. I've tried all the USB ports. I've tried to do it from a USB hub too. I've tried multiple different cables including Apples Lighting cable. I'm getting the same USBMUX Error -79.

I have made sure usbmuxd is updated. I have removed Face ID and my passcode. All of which generates Error -79. Anthing other ideas?
